dev-python/lyricwikia/lyricwikia-0.1.11.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python API to get song lyrics from LyricWikia"
+HOMEPAGE="https://github.com/enricobacis/lyricwikia"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/pytest-runner[${PYTHON_USEDEP}]"
```

Comment:
Are tests removed now?

Reply:
They are, because one of them fails. What is the policy here? Should I add tests even if they fail?

Reply:
Fixed tests, turns out one of the tests requires internet, this was failing, such tests are apparently not allowed in ebuilds therefore I have patched it out.